src/addresses/Ethereum.sol

```diff
@@ -886,17 +886,44 @@ library Fraxswap {
 
 library FraxtalL1 {
     address internal constant ADDRESS_MANAGER = 0x8c5D64d10394cFa070066e70Ec19E67398b4dABE;
+    address internal constant BASE_FEE_VAULT_RECIPIENT = 0xB1748C79709f4Ba2Dd82834B8c82D4a505003f27;
+    address internal constant BATCH_INBOX = 0xfF000000000000000000000000000000000420fC;
+    address internal constant BATCH_SENDER = 0x6017f75108f251a488B045A7ce2a7C15b179d1f2;
+    address internal constant DEPLOYER = 0xe0d7755252873c4eF5788f7f45764E0e17610508;
     address internal constant FRAXTAL_COMPTROLLER = 0xe0d7755252873c4eF5788f7f45764E0e17610508;
+    address internal constant FINAL_SYSTEM_OWNER = 0xe0d7755252873c4eF5788f7f45764E0e17610508;
     address internal constant DISPUTE_GAME_FACTORY_PROXY = 0x3AA7e876880CB34371c01f1510B1e0A932dCb8C1;
+    address internal constant L1_CROSS_DOMAIN_MESSENGER_IMPL = 0xC6e4ECE5576dC96846bb11eD8910f3dF3CE516b8;
+    address internal constant L1_CROSS_DOMAIN_MESSENGER_OLD_IMPL = 0x31c25985E6b10CF335FC01cf9790aF4890d0505c;
     address internal constant L1_CROSS_DOMAIN_MESSENGER_PROXY = 0x126bcc31Bc076B3d515f60FBC81FddE0B0d542Ed;
     address internal constant L1_ERC721_BRIDGE_PROXY = 0xa9B5Fb84B7aeAF0D51C95DB04a76B1D4738D0eC5;
+    address internal constant L1_ERC721_BRIDGE_OLD_IMPL = 0x23A0478996241F9f966b7054c06FdeB96Ddf3475;
+    address internal constant L1_ERC721_BRIDGE_IMPL = 0x7eE1637C33Ed0DFEE6403f3301B6d404106018e4;
     address internal constant L1_STANDARD_BRIDGE_PROXY = 0x34C0bD5877A5Ee7099D0f5688D65F4bB9158BDE2;
+    address internal constant L1_STANDARD_BRIDGE_OLD_IMPL_ADDR = 0xCC26248B71284B812Ff7825e005560DB01a874C7;
+    address internal constant L1_STANDARD_BRIDGE_IMPL_ADDR = 0x30Cd695249394eED44C576f2caEC5304be6455F9;
+    address internal constant L2_OUTPUT_ORACLE_CHALLENGER = 0xe0d7755252873c4eF5788f7f45764E0e17610508;
+    address internal constant L2_OUTPUT_ORACLE_PROPOSER = 0xFb90465f3064fF63FC460F01A6307eC73d64bc50;
     address internal constant L2_OUTPUT_ORACLE_PROXY = 0x66CC916Ed5C6C2FA97014f7D1cD141528Ae171e4;
+    address internal constant L2_OUTPUT_ORACLE_OLD_IMPL = 0xA30ADd78D5be6B2B55C934666bb4fF2C4429d800;
+    address internal constant L2_OUTPUT_ORACLE_IMPL = 0x6f3CcC8C9DAf8B9b39aDe481213Ff7A626a42B65;
     address internal constant OPTIMISM_MINTABLE_ERC20_FACTORY_PROXY = 0x11FE3be54aC01C13Dd985cE2BdD10eD77e1376cc;
     address internal constant OPTIMISM_PORTAL_PROXY = 0x36cb65c1967A0Fb0EEE11569C51C2f2aA1Ca6f6D;
+    address internal constant OPTIMISM_PORTAL_OLD_IMPL_ADDR = 0x59B6903044Fa9439a5D621D3605789A6E654b1D6;
+    address internal constant OPTIMISM_PORTAL_IMPL_ADDR = 0x47Ec03B67ba4f3C71a4320D2947f2173E0B45a23;
+    address internal constant P2P_SEQUENCER_ADDRESS = 0xc88138f5c82DD1bD327708C7F1c15E44Ce7FdA0C;
```

Comment:
Is this address correct? It does not have any txs on [mainnet](https://etherscan.io/address/0xc88138f5c82DD1bD327708C7F1c15E44Ce7FdA0C).

Reply:
@FortisFortuna

Reply:
I saw that. Let me ask Alex.

Reply:
Do we need the implementations? Because. there will be multiple "old" ones in the future as we upgrade them

Btw yes they're correct https://github.com/FraxFinance/fraxtal-deployments/blob/master/mainnet/2024-04-24-upgrade-to-1.3.0/.env.sample#L19-L21

Reply:
Sequencer is 0xc88138f5c82DD1bD327708C7F1c15E44Ce7FdA0C and yeah it's normal it doesn't have txs

Reply:
> Do we need the implementations? Because. there will be multiple "old" ones in the future as we upgrade them

I think so, it should be a helpful reference to track our upgrade path publicly. Otherwise if we are to refer to past versions we would have to sort through `fraxtal-deployments`.